package.json

```diff
@@ -1,190 +1,189 @@
 {
-  "name": "roo-cline",
-  "displayName": "Roo Cline",
-  "description": "Autonomous coding agent right in your IDE, capable of creating/editing files, running commands, using the browser, and more with your permission every step of the way.",
-  "publisher": "RooVeterinaryInc",
-  "version": "2.1.6",
-  "files": [
-    "bin/roo-cline-2.1.6.vsix",
-    "assets/icons/icon_Roo.png"
-  ],
```

Comment:
I'll let @ColemanRoo speak to this one - I'm not that familiar with the packaging setup

Reply:
I don't see any reason why these can't be removed, they were used when we were packaging this for install from an NPM repository. 
It was on my list to remove them.